### PR TITLE
docs(bootstrap): add statement about Bootstrap v5 support

### PR DESCRIPTION
**Pull request by silviyaboteva**
Resolves: https://github.com/telerik/kendo-themes/issues/2730

//cc @nrch

### CODE REVIEW

File: packages/bootstrap/docs/index.md [hunk 1]
```diff
@@ -14,6 +14,19 @@ The Kendo UI for Angular Bootstrap theme is part of the Kendo UI for Angular lib
 The KendoReact Bootstrap theme is part of the KendoReact library of React UI components. It is distributed through NPM under the [kendo-theme-bootstrap package](https://www.npmjs.com/package/@progress/kendo-theme-bootstrap).
 {% endplatform_content %}
 
+## Bootstrap 5 Support
+
+{% platform_content angular %}
+As of 4.40.0 version, the Kendo UI for Angular Bootstrap theme officially supports Bootstrap v5. More information about the migration process and changes could be found in [this issue in our kendo-themes GitHub repository](https://github.com/telerik/kendo-themes/issues/2154). To see the new theme in action, go to any of the KendoReact demos and select Bootstrap from the theme chooser drop-down.
```
Comment (JoomFX) [inline packages/bootstrap/docs/index.md:20]: > go to any of the KendoReact demos

I believe this should be "go to any of the "Kendo UI for Angular demos"

File: packages/bootstrap/docs/index.md [hunk 2]
```diff
@@ -14,6 +14,19 @@ The Kendo UI for Angular Bootstrap theme is part of the Kendo UI for Angular lib
...
+
+Additionally, the Bootstrap theme exposes the older v3 and v4 appearances though theme swatches that could be found in [the kendo-themes GitHub repository](https://github.com/telerik/kendo-themes/tree/master/packages/bootstrap/lib/swatches).
```
Comment (JoomFX) [inline packages/bootstrap/docs/index.md:22]: there is a typo in `through` :)

File: packages/bootstrap/docs/index.md [hunk 3]
```diff
@@ -14,6 +14,19 @@ The Kendo UI for Angular Bootstrap theme is part of the Kendo UI for Angular lib
...
+{% endplatform_content %}
+{% platform_content react %}
+As of 4.40.0 version, the KendoReact Bootstrap theme officially supports Bootstrap v5. More information about the migration process and changes could be found in [this issue in our kendo-themes GitHub repository](https://github.com/telerik/kendo-themes/issues/2154). To see the new theme in action, go to any of the KendoReact demos and select Bootstrap from the theme chooser drop-down.
+
+Additionally, the Bootstrap theme exposes the older v3 and v4 appearances though theme swatches that could be found in [the kendo-themes GitHub repository](https://github.com/telerik/kendo-themes/tree/master/packages/bootstrap/lib/swatches).
```
Comment (JoomFX) [inline packages/bootstrap/docs/index.md:27]: again, a typo in `through` :)

**Review by JoomFX — CHANGES_REQUESTED** [review]

**Comment by joneff** [comment]
@silviyaboteva I've simplified the wording a bit and clarified that the theme does not support Bootstrap 5, but rather depends on it.

**Comment by silviyaboteva** [comment]
@joneff @JoomFX can you double-check now?

**Review by joneff — APPROVED** [review]

**Review by Xizario — APPROVED** [review]
perfect

**Pull request merged**